src/helpers/files.js

```diff
@@ -187,6 +188,8 @@ export const FilesWorkspaceHeader = new Header({
 
 	render(el, folder, view) {
 		addMenuRichWorkspace()
+		console.log(el, folder, view)
+		const hasRichWorkspace = folder.attributes['rich-workspace-file']
```

Comment:
@skjnldsv I'm having some trouble receiving the additionally registered properties on the initial page load.

Registration happens here: https://github.com/nextcloud/text/pull/4776/files#diff-6653fe3a8ac35dad2e28ef2134d921749c4334d4b81a593f80ae312cde95df4eR38

Is there anything in terms of loading order that can be done to fix that? When navigating to a sub folder this works fine.